opensubtitles/errors.go

```diff
@@ -0,0 +1,18 @@
+package opensubtitles
+
+import (
+	"fmt"
+	"net/http"
+)
+
+type ErrorResponse struct {
```

Comment:
Codacy found an issue: [exported type ErrorResponse should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)